modules/scanning-images-by-using-secured-clusters.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="scanning-images-by-using-secured-clusters_{context}"]
+= Scanning images by using secured clusters
+
+To scan images by using the secured clusters instead of the Central services, you can use the delegated image scanning feature.
+
+A new delegated scanning configuration specifies the registries from which you can delegate image scans. For images that Sensor observes, you can use the delegated registry configuration to delegate scans from no registries, all registries, or specific registries.
```

Comment:
I won't hold up merging but for the future, we try to avoid using "new" in anything except release notes, since after a few releases, it won't be new.